web/src/components/Toolbar.vue

```diff
@@ -4,8 +4,8 @@
     color="primary"
     dark
     src="../assets/background.png"
```

Comment:
I think the discontinuity of the purple background between the hero image and the Toolbar looks a bit weird. How about removing the background from the toolbar and letting it be transparent on top of the hero image?